src/Main.java

```diff
@@ -1,15 +1,58 @@
-//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
-// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
+    public Main() {
+    }
+
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.print"Спокойной ночи мир!!!";
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        System.out.println("Ураааааа!!!!");
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+        dog *= 2.0;
+        cat *= 2.0;
+        paper *= 2;
+        dog -= 3.5;
+        --cat;
+        paper -= 7639;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+        var friend = 19;
+        System.out.println(friend);
+        friend += 2;
+        System.out.println(friend);
+        friend /= 7;
+        var frog = 3.5;
+        frog *= 10.0;
+        System.out.println(frog);
+        frog /= 3.5;
+        System.out.println(frog);
+        frog += 4.0;
+        System.out.println(frog);
+        System.out.println("-------------------------------------------------------------------------------------------");
+        var weightBoxer1 = 78.2;
+        var weightBoxer2 = 82.7;
+        var weightAll = weightBoxer1 + weightBoxer2;
+        System.out.println(weightAll);
+        var difference = weightBoxer2 % weightBoxer1;
+        System.out.println(difference);
+        System.out.println("-------------------------------------------------------------------------------------");
+        var allTime = 640;
+        var workerTime = 8;
+        var allWorkers = 80;
+        System.out.println("Если в компании работает " + allWorkers + " человек, то всего " + allTime + " часов работы может быть поделено между сотрудниками");
+        System.out.println("Всего работников в компании "+allTime/workerTime+" человек");
+        allWorkers=allWorkers+94;
+        System.out.println("Если в компании работает "+allWorkers+" человек,то всего "+allTime+" часов работы может быть поделено между сотрудниками");
```

Comment:
чтобы узнать общее число часов надо кол-во сотрудников УМНОЖИТЬ на кол-во часов на одного сотрудника
```suggestion
        System.out.println("Если в компании работает "+allWorkers+" человек,то всего "+allWorkers*workerTime+" часов работы может быть поделено между сотрудниками");
```